src/plugins/saved_objects_finder/public/finder/saved_object_finder.tsx

```diff
@@ -131,22 +122,34 @@ export class SavedObjectFinderUi extends React.Component<
       return col;
     }, []);
 
-    const perPage = this.props.services.uiSettings.get(LISTING_LIMIT_SETTING);
-    const hasReference = this.props.services.savedObjectsManagement.getTagFindReferences({
-      selectedTags,
-      taggingApi: this.props.services.savedObjectsTagging,
-    });
+    const perPage = uiSettings.get(LISTING_LIMIT_SETTING);
     const params: FindQueryHTTP = {
-      type: visibleTypes ?? Object.keys(metaDataMap),
+      type: Object.keys(metaDataMap),
       fields: [...new Set(fields)],
-      search: queryText ? `${queryText}*` : undefined,
       page: 1,
       perPage,
       searchFields: ['title^3', 'description', ...additionalSearchFields],
       defaultSearchOperator: 'AND',
-      hasReference: hasReference ? JSON.stringify(hasReference) : undefined,
     };
-    const response = (await this.props.services.http.get('/internal/saved-objects-finder/find', {
+
+    const { queryText, visibleTypes, selectedTags } = savedObjectsManagement.parseQuery(
```

Comment:
Ah, fair point. This is a leftover from when I wanted to make `savedObjectsManagement` an optional dependency, so the parameters could have been assigned with or without it. But since now it's a required dependency, will change it 👍